rust/operator-binary/src/backend/k8s_search.rs

```diff
@@ -109,4 +89,55 @@ impl SecretBackend for K8sSearch {
                 .collect(),
         ))
     }
+
+    async fn get_qualified_node_names(
+        &self,
+        selector: &SecretVolumeSelector,
+    ) -> Result<Option<HashSet<String>>, Self::Error> {
+        if selector.scope.contains(&SecretScope::Node) {
+            let label_selector = build_label_selector_query(selector, None)?;
+            Ok(Some(
+                self.client
+                    .list::<Secret>(
+                        Some(__self.search_ns_for_pod(selector)),
+                        &ListParams::default().labels(&label_selector),
+                    )
+                    .await
+                    .context(SecretQuerySnafu)?
+                    .into_iter()
+                    .filter_map(|secret| secret.metadata.labels?.remove(LABEL_SCOPE_NODE))
```

Comment:
Is it worth it creating a Snafu Error for this? I'm not certain if this field can be None

Reply:
I'm not sure that the Go people consider this a stable interface, since Go generally treats "empty dict" and `null`/`undefined` as the same (unless specifically requested to be a pointer).